### PR TITLE
chore(tests): split up e2e tests - tfm case

### PR DESCRIPTION
**Pull request by ttbarnes**
## Introduction :pencil2:
This PR splits up the "TFM - case" E2E tests into child runners/directories.

Prior to this change, these tests took ~15 minutes to run.

Each directory now takes ~1 minute - 1.5 minutes to execute, with 1x test taking the most amount of time at 2.5 minutes.

This is part of a series of E2E test splitting to reduce E2E GHA time.

## Resolution :heavy_check_mark:
- Split up some root "TFM - case" E2E tests.
- Split up root "TFM - case - parties" E2E tests.
- Split up root "TFM - case - tasks" E2E tests.
- Update `test.yml`.

## Miscellaneous :heavy_plus_sign:
- Fix a typo (`bankResponse` => `bankResponses`).
- Remove an unnecessary `.clear()`.

### CODE REVIEW

**Review by abhi-markan — DISMISSED** [review]
LGTM 👍🏼

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -606,12 +606,22 @@ jobs:
       # Single dimension matrix
       matrix:
         spec: [
-            'case/*.spec.js',
+
```
Comment (AlexBramhill) [inline .github/workflows/test.yml:609]: Free new line

File: trade-finance-manager-api/src/v1/controllers/send-deal-submit-emails.js [hunk 2]
```diff
@@ -142,8 +142,8 @@ const sendAinMinAcknowledgement = async (deal) => {
       // send a copy of the email to PIM
       const pimEmailResponse = await sendTfmEmail(templateId, pimEmail, emailVariables, deal);
       // send a copy of the email to the bank's general email address
-      const bankResponse = await Promise.all(bankEmails.map((email) => sendTfmEmail(templateId, email, emailVariables, deal)));
-      return { makerEmailResponse, pimEmailResponse, bankResponse };
+      const bankResponses = await Promise.all(bankEmails.map((email) => sendTfmEmail(templateId, email, emailVariables, deal)));
```
Comment (AlexBramhill) [inline trade-finance-manager-api/src/v1/controllers/send-deal-submit-emails.js:145]: Nice job

File: e2e-tests/tfm/cypress/e2e/journeys/case/tasks/AIN/tasks-AIN.spec.js [hunk 3]
```diff
@@ -1,9 +1,9 @@
-import relative from '../../../relativeURL';
-import partials from '../../../partials';
-import pages from '../../../pages';
-import MOCK_DEAL_AIN from '../../../../fixtures/deal-AIN';
-import * as MOCK_USERS from '../../../../../../e2e-fixtures';
-import { BANK1_MAKER1, ADMIN } from '../../../../../../e2e-fixtures';
+import relative from '../../../../relativeURL';
```
Comment (AlexBramhill) [inline e2e-tests/tfm/cypress/e2e/journeys/case/tasks/AIN/tasks-AIN.spec.js:1]: I'm not seeing these files as moved -- Were these test files previously missed and not runnning/why were these passing prior to this pr?

**Review by AlexBramhill — COMMENTED** [review]
lgtm

File: .github/workflows/test.yml (hunk 1 above)
Comment (abhi-markan) [reply .github/workflows/test.yml:609]: Good spot

Comment (ttbarnes) [reply .github/workflows/test.yml:609]: Whoops, excellent spot

File: e2e-tests/tfm/cypress/e2e/journeys/case/tasks/AIN/tasks-AIN.spec.js (hunk 3 above)
Comment (ttbarnes) [reply e2e-tests/tfm/cypress/e2e/journeys/case/tasks/AIN/tasks-AIN.spec.js:1]: They have been moved, can see in the file name in the git differences above - just moved into a child directory.

All these tests were running before, but they were all under 1x single runner `/case/*.spec.js`

**Review by abhi-markan — APPROVED** [review]

**Pull request merged**